composer.json

```diff
@@ -46,7 +35,9 @@
         "behat/mink-selenium-driver": "*",
         "behat/mink-sahi-driver": "*",
         "behat/mink-selenium2-driver": "*",
-        "ezsystems/behatbundle": "@dev"
+        "ezsystems/behatbundle": "@dev",
+        "sensio/generator-bundle": "~2.3",
+        "egulias/listeners-debug-command-bundle": "~1.9"
```

Comment:
So, I vote for having composer dev-mode be the default for all site-development installations. Mian advantage: this might encourage site devs to write unit tests :-D